### ISSUE TITLE
bug(youtube): full screen landscape does not autorotate

### ISSUE THREAD

**Issue by Cats1337** [ReVanced/revanced-patches#800]
### Type

Cosmetic

### Bug description

Fullscreen Landscape Rotation doesn't work correctly. When entering fullscreen it defaults to one direction and can only be changed by using Auto-Rotate. The issue is fixed when spoofing version to 16.xx instead of 17.xx but only once after fresh-install and patch, and upon restart after the issue arises again and persists.

Auto Rotate is off and doesn't need to be on when using stock for it to work.

### Steps to reproduce

1. Patch ReVanced if not already done, duh.
2. Turn Auto-Rotate off, if it's on, psycho.
3. Open ReVanced YouTube
4. Select a video
5. Go into fullscreen
6. Turn phone 180∘
-= Does not rotate in another direction. Leaving video upsidedown.

### Relevant log output

```shell
No log
```

### Solution

Possible solution could be enabling Auto-Rotate when entering fullscreen so that the video snaps to the direct viewing orientation/direction

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [x] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
